cmd/tchaik/ui/static/js/src/stores/PlaylistStore.js

```diff
@@ -298,7 +298,7 @@ function remove(itemIndex, path) {
   setPlaylistItems(items);
   var current = getPlaylistCurrent();
   if (removedItem) {
-    if (current.item === itemIndex) {
+    if (current === null || current.item === itemIndex) {
```

Comment:
This should be `if (current && current.item == itemIndex) {`

There is no need to check if `current` is null and set it to null again.